src/vector/platform/IPCManager.ts

```diff
@@ -53,7 +53,12 @@ export class IPCManager {
         const callbacks = this.pendingIpcCalls[payload.id];
         delete this.pendingIpcCalls[payload.id];
         if (payload.error) {
-            callbacks.reject(payload.error);
+            // `seshat.ts` sends a JavaScript object with a `message` property. Turn it into a proper Error.
+            let error = payload.error;
+            if ((error as any).message) {
```

Comment:
```suggestion
            if ("message" in error) {
```

or `typeof error === "object"` are more correct here, using `any` isn't great